recipes/cocotb/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cocotb" %}
+{% set version = "1.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cocotb-{{ version }}.tar.gz
+  sha256: 1f390bb6468d70e04acebeb9ff05b54be42cbbbbbb3bd6c75d1ec0a36cb97e23
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  missing_dso_whitelist:
+    - '*/libgcc_s.so.1'
+    - '*/libstdc++.so.6'
+  skip:
+    - true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - setuptools
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - cocotb
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.cocotb.org
+  summary: cocotb is a coroutine based cosimulation library for writing VHDL and Verilog testbenches in Python.
+  license: BSD-4-Clause & BSD-3-Clause
+  license_file:
+    - LICENSE
+    - cocotb/_vendor/tcl/license.terms
```

Comment:
This is incomplete. You need to include the license information for ALL of the revendored source code.

https://github.com/cocotb/cocotb/tree/master/cocotb/_vendor